Scenarios/StorageMigrationService/Readme.md

```diff
@@ -0,0 +1,85 @@
+<!-- TOC -->
+
+- [About the scenario](#about-the-scenario)
+    - [Description](#description)
+    - [Scenario requirements](#scenario-requirements)
+- [LabConfig.ps1](#labconfigps1)
+- [The lab](#the-lab)
+
+<!-- /TOC -->
+
+## About the scenario
+
+FYI this is a work in progress
+
+## Description
+* In this scenario you will create a Storage Migration Service lab
+* It will create the default DC, WS2019 SMS server, one or several older servers to migrate storage off
+* This lab uses several diffrent Images from 2008R2 to 2019 RS5
+* Labscript takes 5-10 minutes to finish (dependins what hardware is used)
+
+
+## Scenario requirements
+
+* Windows 10 1511 with enabled Hyper-V or Windows 10 1607+ 
+* 8+ GB RAM is required for this scenario
+* SSD (with HDD it is really slow, barely usable)
+* Create OS images with the OS you need with the Convert-WindowsImage.ps1 file
+* Internet connection is required to Install Chrome on DC. So external switch is created. On Win10 there is a bug that internet get's slow on wireless network.
+
+##To create Windows Images
+````PowerShell
+#Run command to enable functions the file is located under \Tools\ where the prereq.ps1 downloaded files
+. .\Convert-WindowsImage.ps1
+
+#Create Base VHD/VHDX files on the diffrent type of OS you need.
+Convert-WindowsImage -SourcePath "2008R2 ISO Path" -Edition Enterprise -DiskLayout BIOS -VHDFormat VHD -VHDPath C:\HyperV\MigrationLab\ParentDisks\Win2008R2.vhd -SizeBytes 60GB -Passthru
```

Comment:
@jantorep Why not VHDx here? :)

Reply:
2008R2 can't be gen2 so need to be Gen1. Could not get it to work with vhdx :)